src/Sulu/Bundle/AdminBundle/Resources/js/components/Tabs/Tab.js

```diff
@@ -1,25 +1,83 @@
 // @flow
 import React from 'react';
+import type {ElementRef} from 'react';
+import {action, observable} from 'mobx';
+import {observer} from 'mobx-react';
 import classNames from 'classnames';
+import debounce from 'debounce';
 import tabStyles from './tab.scss';
 
 type Props = {
     children: string,
     index?: number,
     onClick?: (index: ?number) => void,
     selected: boolean,
+    setWidth?: (index: ?number, width: number) => void,
+    small: boolean,
 };
 
-export default class Tab extends React.PureComponent<Props> {
+const DEBOUNCE_TIME = 200;
+
+@observer
+class Tab extends React.Component<Props> {
+    @observable selected = false;
```

Comment:
This is needed to show animations on freshly mounted tabs, if they are selected

Reply:
the bottom-border selected indicator, which grows from 0px to 100% or something like that.